jerry-port/default/include/jerryscript-port-default.h

```diff
@@ -40,6 +40,8 @@ void jerry_port_default_set_log_level (jerry_log_level_t level);
 void jerry_port_default_jobqueue_init (void);
 jerry_value_t jerry_port_default_jobqueue_run (void);
 
+void jerry_port_set_instance (jerry_instance_t *instance_p);
```

Comment:
this is not a generic jerry port API function but an addition by the default port implementation. all such functions should be prefixed by jerry_port_default_ (not by jerry_port_) to highlight that they are not available in all ports